components/02-elements/price/Price.html

```diff
@@ -1,21 +1,31 @@
-<div class="price">
+<div class="price price__wrapper">
```

Comment:
Do we need `price__wrapper`. IMO we can set styles for `.price` class

Reply:
Right, but mostly is created for BEM semantic purposes - `--large` modifier. As far as I checked designs same values occurs on list, cart and minicart views. `--large` is created for product page view where font size is increased.

Let me know if I  should remove this and different font sizes should be maintained on the parent components level.

Reply:
Right, It was my misunderstood regarding (mentioned while ago) BEM's modifier naming convention for blocks. Will be fixed :+1: